src/scales/linearScale.ts

```diff
@@ -25,6 +25,14 @@ namespace Plottable.Scales {
       return this._d3Scale(value);
     }
 
+    public scaleTransformation(value: number) {
+      return this.scale(value);
+    }
+
+    public getTransformationDomain() {
+      return this.domain() as [number, number];
```

Comment:
nit: might make sense to have this.domain() return a `[number, number]` type directly